codebasin/walkers/tree_printer.py

```diff
@@ -26,12 +26,12 @@ def __print_nodes(self, node, level):
         """
         association = self._node_associations[node]
         if association:
-            platform = ', '.join(association)
+            platform = ", ".join(association)
         else:
-            platform = ''
+            platform = ""
 
-        spacing = '  ' * (level)
-        print('{}{} -- Platforms: {}'.format(spacing, node, platform))
+        spacing = "  " * (level)
+        print(f"{spacing}{node} -- Platforms: {platform}")
```

Comment:
Is this `print` statement intentional? I think we discussed offline that you actually did intend to `print` as a means of getting the user's attention, in contrast to logging.

Strictly speaking I guess the "best practice" would be to add a `StreamHandler` to `log`, but I'm okay with this being a one off thing.

Reply:
I actually can't find any usage of `TreePrinter` at all...  Can you double-check?

Either way, I think this is probably something that we would want to remove in 2.0.0.  I _think_ it has to wait for a major release -- as tempted as I am to say that our "public API" is actually the command-line interface, it's theoretically possible that somebody somewhere is importing the `codebasin` package and then using this class.